rules.md

```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
    * “ANSWERED IN TASK DESCRIPTION (EXPLICITLY OR IMPLICITLY)” – The task description contains sufficient information. The contestant should read it again carefully.
    * “INVALID QUESTION” – The question is most likely not phrased so that a yes/no answer would be meaningful. The contestant is encouraged to rephrase the question.
    * “NO COMMENT” – The contestant is asking for information that the Scientific Committee cannot give.
+
+<br>
+
+### F. Submission and Access
```

Comment:
Maybe add something like this as the last point:

- For the sake of respecting fairness of this newly-introduced online contest format, each contestant is expected to show a good sportsmanship by following the above rules honestly.

Reply:
Done.